lib/port.c

```diff
@@ -338,10 +338,8 @@ static struct port *getttyuser (const char *tty, const char *user)
 	setportent ();
 
 	while ((port = getportent ()) != NULL) {
-		if (   (0 == port->pt_names)
-		    || (0 == port->pt_users)) {
+		if (NULL == port->pt_users)
```

Comment:
What you say is true now, but could become untrue in the future.  Should we put a comment at the top of getportent()?  Keeping the NULL check seems safest to me in the face of future code changes.